.github/workflows/job_pytorch_models_tests.yml

```diff
@@ -140,11 +140,23 @@ jobs:
           USE_SYSTEM_CACHE: False
           OP_REPORT_FILE: ${{ env.INSTALL_TEST_DIR }}/TEST-torch_unsupported_ops.log
 
+      - name: PagedAttention Test
+        if: always()
+        run: |
+          export PYTHONPATH=${MODEL_HUB_TESTS_INSTALL_DIR}:$PYTHONPATH
+        # For now added the precommit tests only
+          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/pytorch/test_sampling.py -v --tb=short -k "test_pa_precommit"
```

Comment:
```suggestion
          export PYTHONPATH=${MODEL_HUB_TESTS_INSTALL_DIR}:$PYTHONPATH
          # For now added the precommit tests only
          python3 -m pytest ${MODEL_HUB_TESTS_INSTALL_DIR}/pytorch/test_sampling.py -v --tb=short -k "test_pa_precommit"
```

Reply:
Seems like this prevents the whole workflow from starting: 
![image](https://github.com/openvinotoolkit/openvino/assets/65596953/485eb014-fbd4-4292-81d2-3eb638d557cc)
in https://github.com/openvinotoolkit/openvino/actions/runs/9258617931/workflow

Reply:
Updated.

Reply:
Fixed.